docs/release-notes/2.22.0.md

```diff
@@ -0,0 +1,44 @@
+# Percona Monitoring and Management 2.22.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">September 21, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
```

Comment:
```suggestion

## Release Highlights:

- **DBaaS (Technical preview):**: The ability for DBaaS users to upgrade already existed Clusters with a newer version of the operator from PMM UI, without any interaction directly with K8s 
```

Reply:
@PaulJacobs-percona, can you check this highlight, please?

Reply:
I recast a little. LGTM